internal/lib/constants.go

```diff
@@ -215,6 +215,7 @@ const (
 	FQDNReusePolicyStrict                      = "strict"
 	FQDNReusePolicyOpen                        = "internamespaceallowed"
 	DefaultPSName                              = "default-backend"
+	ConcurrentUpdateError                      = "the object has been modified"
```

Comment:
whats the full error string that we get while updation? Is this "the object has been modified" string enough to distinguish from other errors(that we know of)?

Reply:
I think this should be good enough. The error string suggests that object needs to be refreshed before making the update call. 
complete error string - `Error occurred while Updating namespace: Operation cannot be fulfilled on namespaces "diff-proj": the object has been modified; please apply your changes to the latest version and try again`
I am not sure if I know what other concurrent update errors are you talking about?